Makefile

```diff
@@ -72,11 +73,12 @@ deps: $(GO_ETHEREUM_DIR) $(SSHA3_DIR)
 		github.com/grpc-ecosystem/go-grpc-prometheus \
 		github.com/go-kit/kit/log \
 		github.com/loomnetwork/yubihsm-go \
+		github.com/loomnetwork/weave-blueprint/src \
 		gopkg.in/check.v1
 	cd $(GO_ETHEREUM_DIR) && git checkout master && git pull && git checkout $(ETHEREUM_GIT_REV)
+	cd $(SRC_DIR) && git checkout split-main && git pull origin split-main
```

Comment:
This doesn't make sense to me, why would you `go get` the current package, and also check it out?

Reply:
I think this is tempory so the plugin can be exported. For this, we need the `split-main` branch built rather than the main one; So it plays nice with `loomchian`.
After its been merged into master for the first time, then this line can be removed.